Form-Controls/index.html

```diff
@@ -14,14 +14,51 @@ <h1>Product Pick</h1>
   </header>
   <main>
   <form>
-    <!-- write your html here-->
-    <!-- try writing out the requirements first-->
+    <div> 
+      <label for="name">Name</label>
+      <input type="text" id="name" placeholder="Enter Your Name" pattern="^[a-zA-Z]+(?:[-' ][a-zA-Z]+)*$" required>
+    </div>
+    <div> 
+      <label for="email">Email</label>
+      <input type="email" pattern="[a-z0-9._%+-]+@[a-z0-9.-]+\.[a-z]{2,4}$" id="email" placeholder="Enter Your e-mail" required>
+    </div>
+    <div>
+    Choose your T-shirt's colour
```

Comment:
In Web Development, there's a very big and important topic called [Accessibility](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility). 

I have linked you a documentation found on MDN Web Docs. For all things Web Development & JavaScript, MDN documentation will generally be your best friend, you can find so much there and the information there is the best! My recommendation would be, if you are unsure of smth, always check MDN docs, it has the most up-to-date and useful information with regards to Web Development (and not only) 😊

So having said this, you can use some [Semantic HTM](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html)L to let the users using screen readers, to let them know how everything is organised on the web site. 

So considering the T shirt colour option for the user, you can technically wrap radio buttons in a <fieldset> with a <legend> tags, to provide more context for the users with screen readers.  

Semantic elements and a better organised HTML structure, can also improve website's SEO, which is a big topic on its own, but that's something for Marketing staff has to deal with.